src/test/unit/FuseERC4626.t.sol

```diff
@@ -77,8 +79,10 @@ contract TestFuseERC4626 is DSTestPlus {
                 mint()
     //////////////////////////////////////////////////////////////*/
 
-    function testMint1(uint128 _shares) public {
-        uint256 shares = uint256(_shares) + 1; // don't fuzz with 0
+    function testMint1(uint256 shares) public {
+        hevm.assume(shares > 0);
+        hevm.assume(shares <= type(uint128).max);
```

Comment:
a more elegant way to bound to type 128.max is to just set the input type to 128 as you had before

Reply:
this construction will waste fuzz cpu on bad inputs